pkg/bitcoin/tx.go

```diff
@@ -0,0 +1,79 @@
+package bitcoin
+
+import (
+	"bytes"
+
+	"github.com/btcsuite/btcd/chaincfg/chainhash"
+	"github.com/btcsuite/btcd/txscript"
+	"github.com/btcsuite/btcd/wire"
+	"github.com/btcsuite/btcutil"
+)
+
+type Input struct {
+	OutputHash  string
+	OutputIndex uint32
+	Script      string
+	Sequence    uint32
+}
+
+type Output struct {
+	Address string
+	Value   int64
+}
+
+type Tx struct {
+	Inputs   []Input
+	Outputs  []Output
+	LockTime uint32
+}
+
+func (s *Service) CreateTransaction(buf *bytes.Buffer, tx *Tx, chainParams ChainParams) error {
```

Comment:
The general idea with the refactoring is being able to test `pkg/bitcoin` without using outside packages like `pb` or `grpc`. This also allowed me to factor out gRPC message specifics (ex, which integer type to use) outside to `adapter.go`.